pages/common/dolt-fetch.md

```diff
@@ -0,0 +1,16 @@
+# dolt fetch
+
+> Download objects and refs from another repository.
+> More information: <https://github.com/dolthub/dolt>.
+
+- Fetch refs and update remote-tracking branches from the default remote named "origin":
```

Comment:
```suggestion
- Fetch the latest changes from the default remote upstream repository (origin):
```

Reply:
solved in 95bb1b6bf7274caad847319db358b771a1736bcf